.env

```diff
@@ -1,6 +1,5 @@
-# I commit this file cause it will not hold any key or secret
 NODE_ENV=development
 PORT=6069
-COMPRESSED_ROOT_PATH="../snapshots/juno/"
+COMPRESSED_ROOT_PATH="../snapshots/"
```

Comment:
i do not like this .env config but its fine for now

future should migrate to config.json and use that with the types